tests/test-env-py38.yml

```diff
@@ -32,6 +32,7 @@ dependencies:
   - lxml # needed for thredds-crawler
   - urlpath
   - datadog
+  - eodatasets3
```

Comment:
Can you please add this new dependency into `apps/dc_tools/setup.cfg` too. 🙏

Reply:
Done